### PR TITLE
Condense environment variable list

### PR DESCRIPTION
**Pull request by oneroyalace**
This PR removes the `ZORKI_AUTH_KEY`/`FORKI_AUTH_KEY` and `ZORKI_SERVER_URL`/`FORKI_SERVER_URL` environment variables and replace them with `HYPATIA_AUTH_KEY`/`HYPATIA_SERVER_URL`. 

It also updates `config/intializers/figaro.rb` to require the presence of all of Zenodotus' environment variables. 

Changes are documented in the developer README.

# Testing
- `rails t`

_Note: before testing, make sure you've set all environment variables. Before merging this PR, we may need to create a shared Tech&Check YouTube API key._

### CODE REVIEW

File: docs/DEVELOPMENT.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
 
 ## Environment Setup
 
+### Environment variables
+To run Zenodotus, you'll need to set the following environment variables. Ask a dev on the team to provide you access to them. 
+- `INSTAGRAM_USER_NAME`
+- `INSTAGRAM_PASSWORD`
+- `FACEBOOK_EMAIL`
+- `FACEBOOK_PASSWORD`
```
Comment (reefdog) [inline docs/DEVELOPMENT.md:10]: Do we actually need the IG and FB credentials here in Zenodotus? I thought the task of scraping those were shunted over to Hypatia?

File: docs/DEVELOPMENT.md [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+- `TWITTER_BEARER_TOKEN`
+- `YOUTUBE_API_KEY`
```
Comment (reefdog) [inline docs/DEVELOPMENT.md:12]: Same question as above re: whether Zeno needs the YouTube key or just Hypatia.

**Review by reefdog — COMMENTED** [review]
This seems to work great, and I was able to fire up the app and run it and such once I'd added the new required vars.

However, I did leave a few inline questions about whether we actually want/need to require the IG, FB, and YT credentials here in Zeno. Isn't the task of scraping those shunted off to Hypatia? Why do we need those configged here?

File: docs/DEVELOPMENT.md (hunk 1 above)
Comment (oneroyalace) [reply docs/DEVELOPMENT.md:10]: Thaaaat's a great point.

Comment (oneroyalace) [reply docs/DEVELOPMENT.md:10]: No we do not need them

File: docs/DEVELOPMENT.md (hunk 2 above)
Comment (oneroyalace) [reply docs/DEVELOPMENT.md:12]: You are once again correct :)

**Review by reefdog — APPROVED** [review]
Huzzah! This looks great, thanks for removing those unused keys.

**Pull request merged**